src/Akka.Persistence.Sql.Linq2Db.Tests/Sqlite/MSSQLiteJournalSpec.cs

```diff
@@ -7,6 +7,12 @@
 
 namespace Akka.Persistence.Sql.Linq2Db.Tests
 {
+    public class MSSQLiteNativeConfigSpec : MSSQLiteJournalSpec
```

Comment:
I noticed that we didn't check all SQL against native config in *actual* test. Some of the benches, however, covered the scenario. So, this never caused us to ship broken code, but now we have a more clear suite for ourselves and contributors to make sure everything works the way they expect.